--- libosso-1.20/src/Makefile.am.orig	2006-10-10 12:46:34.000000000 +0200
+++ libosso-1.20/src/Makefile.am	2007-06-03 20:59:01.247276880 +0200
@@ -1,5 +1,5 @@
-CFLAGS = $(OSSO_CFLAGS) $(GLIB_CFLAGS) $(DBUS_CFLAGS) $(GNOMEVFS_CFLAGS) -I$(top_srcdir) -I$(top_srcdir)/src \
-		 -DOSSO_CTRLPANELPLUGINDIR=\"${prefix}/lib/hildon-control-panel\"
+AM_CFLAGS = $(OSSO_CFLAGS) $(GLIB_CFLAGS) $(DBUS_CFLAGS) $(GNOMEVFS_CFLAGS) -I$(top_srcdir) -I$(top_srcdir)/src \
+		 -DOSSO_CTRLPANELPLUGINDIR=\"${libdir}/hildon-control-panel\"
 
 INCLUDES = \
 	-DPREFIX=\"${prefix}\"
--- libosso-1.20/configure.ac.orig	2006-10-10 12:46:34.000000000 +0200
+++ libosso-1.20/configure.ac	2007-06-03 20:58:24.385176228 +0200
@@ -20,12 +20,12 @@ AC_ARG_ENABLE(debug, 
 ## g_array_index gives bogus warnings for -Wcast-align on ARM, so we
 ## don't us that warning as well.
 
-WFLAGS="-Werror -Wall -Wmissing-prototypes -std=c99"
+WFLAGS="-Wall -Wmissing-prototypes -std=c99"
 if test x${libosso_use_debug} = xyes ; then
     AC_DEFINE(DEBUG,1,[Build extra debug code])
-    OSSO_CFLAGS="-g -ansi -D_BSD_SOURCE -rdynamic $WFLAGS"
+    OSSO_CFLAGS="-ansi -D_BSD_SOURCE -rdynamic $WFLAGS"
 else
-    OSSO_CFLAGS="-g -ansi -D_BSD_SOURCE -O2 -rdynamic $WFLAGS"
+    OSSO_CFLAGS="-ansi -D_BSD_SOURCE -rdynamic $WFLAGS"
 fi
 
 AC_SUBST(OSSO_CFLAGS)
