--- libosso-2.35/configure.ac.orig	2019-10-02 17:11:56.779069364 +0200
+++ libosso-2.35/configure.ac	2019-10-02 17:13:14.401982178 +0200
@@ -46,9 +46,9 @@
 WFLAGS="-Wall -Wmissing-prototypes -std=c99"
 if test x${libosso_use_debug} = xyes ; then
     AC_DEFINE([LIBOSSO_DEBUG],1,[Build extra debug code])
-    OSSO_CFLAGS="-g -ansi -D_DEFAULT_SOURCE -D_GNU_SOURCE -rdynamic $WFLAGS"
+    OSSO_CFLAGS="-ansi -D_DEFAULT_SOURCE -D_GNU_SOURCE -rdynamic $WFLAGS"
 else
-    OSSO_CFLAGS="-g -ansi -D_DEFAULT_SOURCE -D_GNU_SOURCE -O2 -rdynamic $WFLAGS"
+    OSSO_CFLAGS="-ansi -D_DEFAULT_SOURCE -D_GNU_SOURCE -rdynamic $WFLAGS"
 fi
 
 AC_SUBST([OSSO_CFLAGS])
